src/Raven.Studio/typescript/viewmodels/database/status/statistics.ts

```diff
@@ -42,20 +42,26 @@ class statistics extends viewModelBase {
     compositionComplete() {
         super.compositionComplete();
 
-        const self = this;
-        $('.stats .js-size-tooltip').tooltip({
+        $('.stats .size-tooltip').tooltip({
             container: "body",
             html: true,
             placement: "right",
             title: () => {
                 return `Data: <strong>${this.stats().dataSizeOnDisk}</strong><br />
                 Temp: <strong>${this.stats().tempBuffersSizeOnDisk}</strong><br />
-                Total: <strong>${this.stats().totalSizeOnDisk}</strong>
-                `
+                Total: <strong>${this.stats().totalSizeOnDisk}</strong>`
             }
         });
 
-        popoverUtils.longWithHover($(".js-identities-header"),
+        const cvTooltip = this.stats().databaseChangeVector.map(cv => `<small>${cv.fullFormat}</small>`)
+            .join("<br>");
+
+        popoverUtils.longWithHover($(".cv-tooltip"),
+            {
+                content: `<div>${cvTooltip}</div>`
+            });
+        
+        popoverUtils.longWithHover($(".identities-header"),
```

Comment:
why it doesn't work? Old selector seems to be valid.

Reply:
Yes, old selector was valid.

I removed it (see line 213) because there is already a selector there (`identities-header`)
that can be used for the tooltip. 
No need to have 2 classes (both `js-identities-header` & `identities-header`) in line 213
So I chose to remove it and use `identities-header` for the tooltip.